target/linux/ath79/nand/base-files/etc/init.d/bootcount

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+
+# SPDX-License-Identifier: GPL-2.0
+
+START=99
+
+start() {
+	. /lib/functions.sh
+
+	case $(board_name) in
+	glinet,gl-ar300m-nand)
+		fw_setenv bootcount 0
```

Comment:
I'm not so familiar with that file, but still it looks to me as this also should be in normal base-files...

Reply:
ibid